KarhooUISDK/CommonUI/UIDimens.swift

```diff
@@ -0,0 +1,126 @@
+public struct UIDimens {
+    public struct Spacing {
+        public static let xxxSmall: CGFloat = 1
+        public static let xxSmall: CGFloat = 2
+        public static let xSmall: CGFloat = 4 //3...5
+        public static let small: CGFloat = 8 //6...10
+        public static let medium: CGFloat = 12 //11...14
+        public static let standard: CGFloat = 16 //15...21
+        public static let large: CGFloat = 24 //22...28
+        public static let xLarge: CGFloat = 32 //29...40
+        public static let xxLarge: CGFloat = 44 //41...56
+        public static let xxxLarge: CGFloat = 64 // 57+
+        public static let marginToEdgeOfScreen: CGFloat = 20
+    }
+    
+    public struct Size {
+        public struct Button {
+            public static let small: CGFloat = 24
+            public static let standard: CGFloat = 44
+            public static let large: CGFloat = 55 //e.g.: the Book button in the checkout screen
+            public static let xlarge: CGFloat = 64
+            public static let xxLarge: CGFloat = 128
+        }
+        
+        public struct Icon {
+            public static let small: CGFloat = 11 //8...14
+            public static let medium: CGFloat = 16 //15...17
+            public static let standard: CGFloat = 22 //18...26
+            public static let large: CGFloat = 32 //27...39
+            public static let xLarge: CGFloat = 44 //44...53
+            public static let xxLarge: CGFloat = 64 //54+
+            public static let logoWidth: CGFloat = 222
+        }
+        
+        public struct Separator {
+            public static let separatorHeight: CGFloat = 1 // 0.5...2
+            public static let largeSeparatorHeight: CGFloat = 3
+            public static let noBorder: CGFloat = 0
+            public static let mediumBorderWidth: CGFloat = 0.5
+            public static let standardBorderWidth: CGFloat = 1
+            public static let largeBorderWidth: CGFloat = 1.25
+        }
+        
+        public struct SafeArea {
+            public static var bottomHeight: CGFloat {
+                get {
+                    if #available(iOS 13.0, *) {
+                        let window = UIApplication.shared.windows.first
+                        return window?.safeAreaInsets.bottom ?? 20
+                    } else {
+                        return 20
+                    }
+                }
+            }
+            public static var topHeight: CGFloat {
+                get {
+                    if #available(iOS 13.0, *) {
+                        let window = UIApplication.shared.windows.first
+                        return window?.safeAreaInsets.top ?? 20
+                    } else {
+                        return 20
+                    }
+                }
+            }
+        }
+        
+        public struct View {
```

Comment:
Maybe just use CGSize everywhere you are using `Size` in name? I suspect this value conforms to side length of the view, therefore we could use, for example, `loadingHudSize: CGSize = .init(height: 22, width: 22)`